pkgs/development/python-modules/combinadics/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib
+  , buildPythonPackage
+  , fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "combinadics";
+  version = "0.0.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-CFbtDgcbrFKEYknegVRSUZbc+jS0OCGN53ZYBAUAFD4=";
+  };
+  doCheck = false;
```

Comment:
Any chance to run the tests (at least partially), or to find out why it won't work?